scripts/create-adaptation-pr.sh

```diff
@@ -141,7 +141,7 @@ echo "### [auto] checkout 'bump/$BUMPVERSION' and merge the latest changes from
 
 git checkout "bump/$BUMPVERSION"
 git pull $MAIN_REMOTE "bump/$BUMPVERSION"
-git merge --no-edit $MAIN_REMOTE/master || true # ignore error if there are conflicts
+git merge --no-rebase --no-edit $MAIN_REMOTE/master || true # ignore error if there are conflicts
```

Comment:
I think the `--no-rebase` should be on `git pull` right? (It's not a valid option for `git merge`).